cms/navigation/models.py

```diff
@@ -0,0 +1,102 @@
+from typing import TYPE_CHECKING, ClassVar
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import FieldPanel
+from wagtail.blocks import CharBlock, ListBlock, PageChooserBlock, StructBlock
+from wagtail.contrib.settings.models import BaseSiteSetting, register_setting
+from wagtail.models import PreviewableMixin
+
+from cms.core.blocks.base import LinkBlock
+from cms.core.fields import StreamField
+
+if TYPE_CHECKING:
+    from django.http import HttpRequest
+
+
+class ThemeLinkBlock(LinkBlock):
+    page = PageChooserBlock(required=False, page_type="themes.ThemePage")
+
+    class Meta:
+        label = "Theme Link"
```

Comment:
as discussed, for consistency, we need to mark for translation any strings that will be displayed to the user

Reply:
Done!